pkg/limayaml/default.yaml

```diff
@@ -31,6 +31,19 @@ memory: "4GiB"
 # Default: "100GiB"
 disk: "100GiB"
 
+network:
+  # The instance can get a routable IP address from the vmnet framework using
+  # https://github.com/AkihiroSuda/vde_vmnet. Both vde_switch and vde_vmnet
+  # daemons must be running before the instance is started. The interface type
+  # (host, shared, or bridged) is configured in vde_vmnet and not lima.
+  vde:
+    # vde_switch socket directory, normally /var/run/vde.ctl
+    # Default: "" (vmnet is not being used)
+    vdeSocketDir: null
```

Comment:
Calling this field “location” might be more consistent with other fields in the YAML.

Or just “vdeSocket” or “vdeURL”, as this path can be non-directory socket when we support PTP mode (switchless mode).

Also, we will be able to support more non-directory  networking types when QEMU upgrades VDE2 to VDEplug4

https://github.com/rd235/vdeplug4/blob/089984200f447abb0e825eb45548b781ba1ebccd/man/libvdeplug.3#L41

Reply:
I went with `url` for future-proofing. I can't bring myself to call it a socket when it is actually a directory containing sockets. 😄